webapp/src/main/java/life/qbic/Application.java

```diff
@@ -67,6 +68,8 @@ private static MessageSubscriber whenUserRegisteredSendEmail(
       }
       try {
         UserRegistered userRegistered = deserialize(message);
+        System.out.println(appContext.getBean(EmailConfirmationLinkSupplier.class)
+            .emailConfirmationUrl(userRegistered.userId()));
```

Comment:
You could add a test to see if it works rather than adding a command-line printout. What do you think in general about a test to see if the link contains all required parts and/or if no user information is shown in the link

Reply:
I do not know what we want to test here. Testing whether the link has the userid at the end is very close to the implementation. So we would test our current implementation rather than the behaviour (which I do not know how to summarise with a test)

I will remove this printout once anything happens to the link (e.g. it is passed into an email)

Reply:
Added some test now to confirm the fixed part of the URL